tests/infrared/metrics-collectd-qdr.yaml.template

```diff
@@ -4,28 +4,47 @@ tripleo_heat_templates:
 
 custom_templates:
     parameter_defaults:
-        CollectdAmqpInterval: 1
-        CollectdDefaultPollingInterval: 1
-        CollectdExtraPlugins:
+        CollectdAmqpInterval: 5
+        CollectdDefaultPollingInterval: 5
+        CollectdDefaultPlugins:
             - cpu
             - df
+            - disk
             - hugepages
+            - intel_rdt
+            - interface
+            - ipmi
+            - load
+            - memory
             - ovs_events
             - ovs_stats
-            - load
+            - processes
+            - unixsock
             - uptime
         DnsServers: ["10.19.42.41","10.11.5.19","10.5.30.160"]
         ExtraConfig:
-            collectd::plugin::cpu::valuespercentage: true
+            collectd::plugin::cpu::interval: 5
             collectd::plugin::cpu::reportbycpu: true
             collectd::plugin::cpu::reportbystate: true
             collectd::plugin::cpu::reportnumcpu: false
+            collectd::plugin::cpu::valuespercentage: true
+            collectd::plugin::df::interval: 300
             collectd::plugin::df::ignoreselected: true
+            collectd::plugin::df::fstypes: ['xfs']
```

Comment:
This ignores all of the bind-mounted config files which each generate 6 metrics but all reflect the values from the rootfs, which is already represented. No other xfs file systems will ever be visible to the collectd container so this should be safe. We may want to explore better ways of reporting on the host file system.